pyproject.toml

```diff
@@ -14,12 +14,56 @@ version_file = "src/jaxsim/_version.py"
 line-length = 88
 
 [tool.isort]
-profile = "black"
 multi_line_output = 3
+profile = "black"
 
 [tool.pytest.ini_options]
-minversion = "6.0"
 addopts = "-rsxX -v --strict-markers"
+minversion = "6.0"
 testpaths = [
-    "tests",
+  "tests",
+]
+
+[tool.ruff]
+exclude = [
+  ".git",
+  ".pytest_cache",
+  ".ruff_cache",
+  ".vscode",
+  ".devcontainer",
+  "__pycache__",
 ]
+preview = true
+
+target-version = "py311"
+
+[tool.ruff.lint]
```

Comment:
```suggestion
# https://docs.astral.sh/ruff/rules/
[tool.ruff.lint]
```

Reply:
Thanks, done in https://github.com/ami-iit/jaxsim/pull/114/commits/f7c63e4ff1a69bb6efd5d40b15544994c938e2ee